types/node-hrx/index.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module "node-hrx" {
```

Comment:
Are these type definitions for node-hrx complete or do they cover only the parts used by the runner ? If they are complete, I suggest upstreaming them to DefinitelyTyped.

Reply:
Honestly it may be more sustainable to just convert node-hrx to TypeScript directly, since afaik this is now one of the main projects using it.